### PR TITLE
Added nullTerminated method to Text, fixed raw method in TextBuffer

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Fixes https://github.com/cogneco/ooc-kean/issues/1059
Fixes https://github.com/cogneco/ooc-kean/issues/1056
@marcusnaslund

### CODE REVIEW

File: source/sdk/Text.ooc [hunk 1]
```diff
@@ -190,6 +190,17 @@ Text: cover {
 		this free(Owner Receiver)
 		result
 	}
+	nullTerminated: func -> This {
```
Comment (simonmika) [inline source/sdk/Text.ooc]: Remove! There is no such thing as a null terminated text.

File: source/sdk/TextBuffer.ooc [hunk 2]
```diff
@@ -20,11 +20,7 @@ import OwnedBuffer
 
 TextBuffer: cover {
 	_backend: OwnedBuffer
-	raw: Char* { get {
-		result := this _backend pointer as Char*
-		this free(Owner Receiver)
-		result
-	}}
+	raw ::= this _backend pointer as Char*
```
Comment (simonmika) [inline source/sdk/TextBuffer.ooc]: Wrong, call to property should always free if called with owner receiver.

File: source/sdk/Text.ooc (hunk 1 above)
Comment (sebastianbaginski) [inline source/sdk/Text.ooc]: So how do you plan to pass a content of Text to the system calls which expects `\0` at the end of char buffer ?

File: source/sdk/TextBuffer.ooc (hunk 2 above)
Comment (sebastianbaginski) [inline source/sdk/TextBuffer.ooc]: This is not valid for property which should return a pointer to internal buffer. Please provide a single use case where this is correct to free a buffer and then return pointer to it.

**Comment by simonmika** [comment]
When you want to no which memory it pointed to. I think it is better to remove `raw` then to make exceptions.

**Pull request closed without merging**